server/routes/api/documents.ts

```diff
@@ -1162,7 +1162,17 @@ router.post("documents.create", auth(), async (ctx) => {
     index,
   } = ctx.body;
   const editorVersion = ctx.headers["x-editor-version"] as string | undefined;
-  assertUuid(collectionId, "collectionId must be an uuid");
+
+  if (parentDocumentId || template) {
+    assertPresent(
+      collectionId,
+      "collectionId is required to create a nested doc or a template"
+    );
+  }
```

Comment:
I'm assuming this is both necessary and sufficient to _not break_ the existing nested doc or template creation flow _requiring_ collection?

Reply:
That sounds right, perhaps we can have nested docs without a collection later but that will require some thought given the ordering data is currently stored on the collection itself